metrics_ci_integration/test/jenkins/resources/jenkins_artifacts_resources.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:ci_integration/jenkins/client/model/jenkins_build_artifact.dart';
+
+/// A class providing a test data for [JenkinsBuildArtifact].
+class JenkinsArtifactsResources {
```

Comment:
```suggestion
class JenkinsArtifactsTestData {
```
? also what if we put it to `test_data` package?